build/root/WORKSPACE

```diff
@@ -1,8 +1,8 @@
 http_archive(
     name = "io_bazel_rules_go",
-    sha256 = "441e560e947d8011f064bd7348d86940d6b6131ae7d7c4425a538e8d9f884274",
-    strip_prefix = "rules_go-c72631a220406c4fae276861ee286aaec82c5af2",
-    urls = ["https://github.com/bazelbuild/rules_go/archive/c72631a220406c4fae276861ee286aaec82c5af2.tar.gz"],
+    sha256 = "e09aa1a3318a292c24b3c97584f2fc3ceb7636ec786e580d1adba1efce320bb7",
+    strip_prefix = "rules_go-b358831ed503659656daa35a361094a1eee5aa60",
+    urls = ["https://github.com/bazelbuild/rules_go/archive/b358831ed503659656daa35a361094a1eee5aa60.tar.gz"],
```

Comment:
Looks like this broke `go_genrule` again, unsurprisingly. I will take a look (unless you want to dive into skylark yourself).